src/index.html

```diff
@@ -1,14 +1,40 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet"
+          href="./style.css"
+    >
+  </head>
+  <body>
+    <div class="main-container">
+      <div class="block first__block">
+        <p>1</p>
+      </div>
+
+      <div class="block second__block">
+        <p>2</p>
+      </div>
```

Comment:
It's better to use modificators
```suggestion
      <div class="block block--1">
        <p>1</p>
      </div>

      <div class="block block--2">
        <p>2</p>
      </div>
```